### DISCUSSION TITLE
Is there a way to grab predictions for players?

### DISCUSSION THREAD

**Discussion by Deom23** [mattdodge/yahoofantasy#34] [Ideas]
I was hoping that player class, or the dump file would have predictions for each player. Is it easy to modify one of those two processes to include the weekly prediction for each player?

**Comment by mattdodge** [comment]
Unfortunately there is not. Yahoo considers their (wildly inaccurate) projections to be a bit of a trade secret, so they do not expose them in any kind of API. At least not one that I have found.

IIRC there might be the ability to access the weekly projections (i.e., what they think your entire team would score for a week) but not the individual player projections.

**Reply by Deom23** [reply]
Yahoo is lame, thanks for the answer and the great package!
